### PR TITLE
Use code blocks for CLI commands in Readme.md

### PR DESCRIPTION
**Pull request by MRDGH2821**
Issue: None

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [ ]  issue was opened to discuss proposed changes before starting implementation. It is important do discuss changes before implementing them (Why should we add it? How should it work? How should it look? Where will it be? ...).
- [ ]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [ ]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [ ]  app version number was not changed.
- [ ]  all new code has tests to ensure against regressions.
- [ ] `npm run lint` reports no offenses.
- [ ] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.
- [ ]  after PR is approved, all commits in it are [squashed](https://gitbetter.substack.com/p/how-to-squash-git-commits)

### Description of the Change

Use code blocks for CLI commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,16 @@ The latest official **installers** and **portable versions** for macOS, Windows
 
 ### macOS
 
-It is recommended to install *Stretchly* with [Homebrew](https://brew.sh/) by running `brew update && brew install --cask stretchly`, see [Application Signing](#application-signing). When upgrading, run `brew update && brew upgrade --cask`. Don't forget to Quit Stretchly, first.
+It is recommended to install *Stretchly* with [Homebrew](https://brew.sh/) by running:
+```zsh
+brew update && brew install --cask stretchly
+```
+See [Application Signing](#application-signing).
```
Comment (hovancik) [inline README.md]: can we move this up? 

It is recommended to install Stretchly with Homebrew  (see Application Signing) by running:

File: README.md [hunk 2]
```diff
@@ -26,7 +26,16 @@ The latest official **installers** and **portable versions** for macOS, Windows
...
+It is recommended to install *Stretchly* with [Homebrew](https://brew.sh/) by running:
+```zsh
+brew update && brew install --cask stretchly
+```
+See [Application Signing](#application-signing). 
+When upgrading, run:
+```zsh
+brew update && brew upgrade --cask
+```
+Don't forget to Quit Stretchly, first.
```
Comment (hovancik) [inline README.md]: Same here, could be: When upgrading, quit Stretchly and then run:

**Review by hovancik — CHANGES_REQUESTED** [review]

**Comment by MRDGH2821** [comment]
Resolved as requested

**Review by hovancik — APPROVED** [review]
Cheers!

**Comment by MRDGH2821** [comment]
Was it code fence or code block ?🤔
Either way, my intent was to make it easier to copy commands.😄

**Comment by hovancik** [comment]
No idea :D

**Pull request merged**